packages/bindings/examples/schema.rs

```diff
@@ -0,0 +1,27 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema, schema_for};
+use terra_bindings::{
+    ExchangeRateResponse, ExchangeRatesResponse, RewardsWeightResponse,
+    SeigniorageProceedsResponse, SimulateSwapResponse, TaxCapResponse, TaxProceedsResponse,
+    TaxRateResponse, TerraMsg, TerraQuery, TobinTaxResponse,
+};
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
```

Comment:
The clean line is missing here (https://github.com/CosmWasm/cosmwasm/blob/v0.8.0-alpha3/contracts/queue/examples/schema.rs#L9-L12)

Reply:
good call

Reply:
good call